app/etc/registration_globlist.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Glob patterns relative to this files parent directory, used by
+ * registration.php to generate a list of includes.
+ */
+return [
+    'code/*/*/cli_commands.php',
```

Comment:
I can't say if a different path makes it more intuitive or not. I give the benefit of the doubt, for my change it was only to streamline based on common base-path, so a technical decision. I found it overall hard(er) to grasp and I don't know if the path was part of it. Perhaps a well written comment would do better than a code / data change. I can also imagine that this might change in the future so I would not suggest to introduce hard rules here.